docs/proposals/20191030-machine-health-checking.md

```diff
@@ -105,14 +104,12 @@ As an operator of a Management Cluster, I want my machines to be self-healing an
 
 ### Implementation Details/Notes/Constraints
 
-#### Machine annotation:
+#### Machine condition:
```

Comment:
Who owns this condition? Machine or MachineHealthCheck?

Ownership is probably on the Machine, while operations are done outside?

Reply:
What do you mean by ownership?

Reply:
Ownership is usually given to the object/controller that defines the contract